tests/utils/testutils.py

```diff
@@ -257,6 +257,11 @@ def setUpClass(cls):
                 _setup_func_app(TESTS_ROOT / script_dir)
                 cls.webhost = start_webhost(script_dir=script_dir,
                                             stdout=cls.host_stdout)
+            if cls.webhost._proc.errors is not None or \
+               cls.webhost._proc.stderr is not None:
+                raise RuntimeError(
+                    'Failed to start WebHost:'
```

Comment:
If both are true then we should print both, errors and stderr.

Reply:
discussed with Varad, this wont catch anything, removed